### ISSUE TITLE
Unable to install this cli tool

### ISSUE THREAD

**Issue by innopreneur** [TomAFrench/create-ocean-app#1]
**Bug Description**
Hey @TomAFrench  
I am unable to install this tool and hence not able to test this bounty submission

**Steps to Reproduce**

1. in your terminal, type `yarn create ocean-app my-ocean-app`

Attaching a screenshot for more details on the issue -

![image](https://user-images.githubusercontent.com/7534044/82206335-ab991f00-9925-11ea-9b4e-a83a9d4243d7.png)

**Comment by TomAFrench** [comment]
Hey @innoprenuer 

That's odd. I have a feeling that this might have something to do with your yarn version as it's a little old (I'm currently on v1.22.4). I've tested this on a couple machines and it works on those.

Could you try out running `yarn create eth-app my-eth-app --template sablier` for me to see if this is `create-ocean-app` specific? If that fails could you try updating yarn and trying again please?

**Comment by innopreneur** [comment]
Hey @TomAFrench sorry for getting it tested late. I didnt got your reply notificiation here somehow. So, I tested with above command (sablier) and got this error -

![image](https://user-images.githubusercontent.com/7534044/85015023-e8567100-b184-11ea-997f-a93d817930f3.png)

**Comment by TomAFrench** [comment]
Yeah in that case I'd recommend you update your yarn version (v1.9.4 is a couple of years old) and try again. I've tested `create-ocean-app` on a couple of machines including a mac and it works fine so I can only assume that that's the issue.

**Comment by innopreneur** [comment]
@TomAFrench I upgraded yarn to latest stable v1.22.4 but still get same error for both commands -

![image](https://user-images.githubusercontent.com/7534044/85279228-b35b5e80-b4a3-11ea-8b7e-a82c7ed0525c.png)
